openshift-ci/build-root/Dockerfile

```diff
@@ -2,5 +2,5 @@
 
 FROM registry.ci.openshift.org/openshift/release:golang-1.15
 
-RUN yum -y install make wget gcc git httpd-tools
+RUN yum -y install make wget gcc git httpd-tools golang
```

Comment:
These changes might affect running integration, no?

Reply:
yeah, that make sense. So I will add it to unit test specific file so that it doesn't affect integration